swarm/network/stream/delivery_test.go

```diff
@@ -219,53 +123,12 @@ func TestStreamerUpstreamRetrieveRequestMsgExchange(t *testing.T) {
 				Peer: node.ID(),
 			},
 		},
-		Expects: []p2ptest.Expect{
-			{
-				Code: 1,
-				Msg: &OfferedHashesMsg{
-					HandoverProof: &HandoverProof{
-						Handover: &Handover{},
-					},
-					Hashes: hash,
-					From:   0,
-					// TODO: why is this 32???
-					To:     32,
-					Stream: stream,
-				},
-				Peer: node.ID(),
-			},
-		},
-	})
-
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	hash = storage.Address(hash1[:])
-	ch = storage.NewChunk(hash, hash1[:])
-	_, err = localStore.Put(context.TODO(), chunk.ModePutUpload, ch)
-	if err != nil {
-		t.Fatalf("Expected no err got %v", err)
-	}
-
-	err = tester.TestExchanges(p2ptest.Exchange{
-		Label: "RetrieveRequestMsg",
-		Triggers: []p2ptest.Trigger{
-			{
-				Code: 5,
-				Msg: &RetrieveRequestMsg{
-					Addr:      hash,
-					SkipCheck: true,
-				},
-				Peer: node.ID(),
-			},
-		},
 		Expects: []p2ptest.Expect{
 			{
 				Code: 6,
 				Msg: &ChunkDeliveryMsg{
-					Addr:  hash,
-					SData: hash,
```

Comment:
How did that work before is beyond me... We don't have any validation for `ChunkDeliveryMsg` ?

Reply:
further up in localstore i guess there is. not on msg level

Reply:
true, there is validation in localstore, i just found this confusing here. now that you mention it, it makes sense.